complete/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+#spring.cloud.gcp.projectId=[YOUR_GCP_PROJECT_ID_HERE]
+#spring.cloud.gcp.credentialsLocation=file:[LOCAL_FS_CREDENTIALS_PATH]
+
+#spring.cloud.gcp.pubsub.subscriber.executorThreads=[NUM_SUBSCRIBER_THREADS]
```

Comment:
what is the recommended setup for the threads? Are these really necessary for the tutorial? 
I would also keep the cloud properties uncommented (one less thing to do for the user to get to a working example)